packages/node_modules/@webex/helper-image/src/index.js

```diff
@@ -42,33 +42,27 @@ export function updateImageOrientation(file, options = {}) {
 * @param {Object} buf
 * @returns {Promise<ExifImage>}
 */
-export function readExifData(file, buf) {
-  return new Promise(async (resolve) => {
-    // For avatar images the file.type is set as image/jpeg, however for images shared in an activity file.mimeType is set as image/jpeg. Handling both conditions.
-    if (
-      file &&
-      (file.type === 'image/jpeg' || file.mimeType === 'image/jpeg')
-    ) {
-      const {
-        Orientation,
-        ExifImageHeight,
-        ExifImageWidth
-      } = await parse(buf, {translateValues: false});
+export async function readExifData(file, buf) {
+  if (
```

Comment:
Let's not lose the comment~
```suggestion
// For avatar images the file.type is set as image/jpeg, however for images shared in an activity file.mimeType is set as image/jpeg. Handling both conditions.
  if (
```